lua/apisix/plugins/proxy-rewrite.lua

```diff
@@ -74,6 +79,17 @@ function _M.rewrite(conf, ctx)
         ctx.var["upstream_upgrade"]    = ctx.var["http_upgrade"]
         ctx.var["upstream_connection"] = ctx.var["http_connection"]
     end
+
+    if conf.headers then
+        for header_name, header_value in pairs(conf.headers) do
+            if header_value == "null" then
+                -- Remove Header
+                core.request.set_header(header_name, "")
```

Comment:
should be `core.request.set_header(header_name, nil)`? I am not sure.

Reply:
currently `core.request.set_header` checks the parameter of `header value`, only `string`, `number`, `boolean` types are allowed. Other types will throw an exception.